docs/articles/documentation/test-api/selecting-page-elements/selector-options.md

```diff
@@ -64,6 +85,7 @@ You can also use the [index](#optionsindex) option to identify a node by its ind
 For details, see [Initializers that Return Multiple Nodes](selectors.md#initializers-that-return-multiple-nodes).
 
 ## options.index
+> Important! ***deprecated*** *Will be removed in next releases. Use [nth()](selectors.md#filter-multiple-dom-nodes) to filter elements by its index.*
```

Comment:
***Deprecated.*** *Use the [nth()](selectors.md#filter-multiple-dom-nodes) method to select a node by its index. The [options.index](#optionsindex) option will be removed in future releases.*